src/OSPSuite.Utility/FileHelper.cs

```diff
@@ -203,30 +203,6 @@ public static bool AreFilesEqual(string fileFullPath1, string fileFullPath2)
          return BitConverter.ToString(hashFile1) == BitConverter.ToString(hashFile2);
       }
 
-      /// <summary>
-      ///    Returns <c>false</c> if the path is read only or do not have access to view the permissions otherwise <c>true</c>
-      /// </summary>
-      public static bool HasWriteAccessToFolder(string folderPath)
-      {
-         try
-         {
-            // Attempt to get a list of security permissions from the folder. 
-            // This will raise an exception if the path is read only or do not have access to view the permissions. 
-            Directory.GetAccessControl(folderPath);
```

Comment:
This does not exist in .NET core. I am not sure where we use this method at all. This will have to be defined in the App code instead